epispot/__init__.py

```diff
@@ -20,6 +20,9 @@
 from . import fitters
 
 # version info
-__version__ = "v2.0.1.24"  # version (v#.#.#)
-__stable__ = True  # True --> stable release, False --> unstable release
+__version__ = "v2.1.1.0"  # version (v#.#.#)
+"""epispot's version info"""
+__stable__ = False
+"""stability of the build: True --> stable release (master), False --> unstable release (nightly)"""
 __repo_data_url__ = 'https://www.github.com/epispot/epispot'  # source repo
+"""URL to GitHub source"""
```

Comment:
Why do you have a docstring for this?

Reply:
So it displays on documentation

Reply:
It's unnecassary

Reply:
It is necessary because it shows how to access the GitHub source URL from the package itself.